src/model/wallet.js

```diff
@@ -47,7 +50,16 @@ export const walletInit = async (seedPhrase = null) => {
 };
 
 export const loadWallet = async () => {
-  const privateKey = await loadPrivateKey();
+  const savedVersion = loadKeychainVersion();
```

Comment:
```suggestion
const savedVersion = await loadKeychainVersion();
```

Reply:
I prefer to use await here and return the promise in the function, but that's just my personal preference